test/lexer.test.cpp

```diff
@@ -37,7 +37,7 @@
 int main (int, char**)
 {
 #ifdef PRODUCT_TASKWARRIOR
-  UnitTest t (1253);
+  UnitTest t (1259);
```

Comment:
What is this about?

Reply:
Oh, I think I need to update that again. It sets the number of tests expected -- part of TAP.